src/Kernel/Helpers/GetEnvironmentVariableHelper.cs

```diff
@@ -0,0 +1,14 @@
+﻿using LT.DigitalOffice.Kernel.Exceptions.Models;
+using System;
+
+namespace LT.DigitalOffice.Kernel.Helpers
+{
+    public static class GetEnvironmentVariableHelper
+    {
+        public static string Get(string key)
```

Comment:
rename Get => GetVariable

Reply:
name of class GetEnvironmentVariableHelper. I think it's clear that we waiting for in the Get method.